index.ts

```diff
@@ -346,12 +330,14 @@ export default class Analytics {
     const req = {
       method: 'POST',
       headers: {
+        Accept: 'application/json, text/plain, */*',
+        'Content-Type': 'application/json;charset=utf-8',
```

Comment:
both headers are required for the express server's body-parser to function properly